### ISSUE TITLE
CiC update 2.10.0 breaks data for entity Flowrate as JSON name was changed to flowRateFiltered

### ISSUE THREAD

**Issue by GtHbbrr** [marcoboers/home-assistant-quatt#81]
## System Information

version | core-2024.1.6
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.11.6
os_name | Linux
os_version | 6.1.63-haos-raspi
arch | aarch64
timezone | Europe/Amsterdam
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 5000
Installed Version | 1.34.0
Stage | running
Available Repositories | 1392
Downloaded Repositories | 5
HACS Data | ok

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 11.4
-- | --
update_channel | stable
supervisor_version | supervisor-2023.12.1
agent_version | 1.6.0
docker_version | 24.0.7
disk_total | 13.9 GB
disk_used | 5.8 GB
healthy | true
supported | true
board | rpi4-64
supervisor_api | ok
version_api | ok
installed_addons | Advanced SSH & Web Terminal (17.0.5), Terminal & SSH (9.8.1)

</details>

<details><summary>Dashboards</summary>

dashboards | 1
-- | --
resources | 2
views | 4
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | 2 February 2024 at 17:07
-- | --
current_recorder_run | 2 February 2024 at 18:07
estimated_db_size | 121.30 MiB
database_engine | sqlite
database_version | 3.41.2

</details>

### Checklist

- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of currently [previous issues](https://github.com/ludeeus/quatt/issues?q=is%3Aissue+label%3A%22Bug%22+)..

### Describe the issue

CiC update 2.10.0 breaks data for entity Flowrate as JSON name was changed to flowRateFiltered

### Reproduction steps

1. open json from CiC
2. see flowrate before and after CiC update 2.10.0
3.
...

### Debug logs

```text
CiC update 2.10.0 breaks data for entity Flowrate as JSON name was changed to flowRateFiltered
```

### Diagnostics dump

CiC update 2.10.0 breaks data for entity Flowrate as JSON name was changed to flowRateFiltered

**Comment by marcoboers** [comment]
Thanks, my CIC is still on 2.9.1. I will fix this tomorrow. 
Did you notice any other changes I should be aware of?

**Comment by GtHbbrr** [comment]
Hi Marco, no nothing else that I noticed.
Please find below a JSON copy on 2.10.0
The new flowrate definition is also moved from flowmeter to qc part of the JSON

{
"boiler":{
"oTtbTurnOnOffBoilerOn":false,
"otFbChModeActive":false,
"otFbDhwActive":false,
"otFbFlameOn":false,
"otFbSupplyInletTemperature":51.089844,"otFbSupplyOutletTemperature":48.898438,
"otTbCH":false},

"flowMeter":{"waterSupplyTemperature":29.005091},

"hp1":{
"getMainWorkingMode":2,
"limitedByCop":false,
"modbusSlaveId":1,
"power":-281.873867,
"powerInput":132.164098,
"silentModeStatus":false,
"temperatureOutside":9.28,
"temperatureWaterIn":29.92,
"temperatureWaterOut":29.68},

"qc":{
"flowRateFiltered":801.0,
"stickyPumpProtectionEnabled":false,
"supervisoryControlMode":2},

"system":{
"hostName":
"CIC-xxx"},

"thermostat":{
"otFtChEnabled":true,
"otFtControlSetpoint":31.0,
"otFtCoolingEnabled":false,
"otFtDhwEnabled":true,
"otFtRoomSetpoint":20.0,
"otFtRoomTemperature":18.726562},

"time":{
"ts":1707138144623,
"tsHuman":"2024-02-05T13:02:24Z"}}

**Comment by Verguldebarman** [comment]
Found a second charge in JSON:

hp1.heatpower has been renamed to hp1.power

**Comment by marcoboers** [comment]
@Verguldebarman hp1.heatpower did never exist. heatPower is a computed property from before power was available in the json, I recommend to switch to use power and powerInput provided by the json instead.

**Comment by marcoboers** [comment]
Fixed the flowrate in [0.5.0](https://github.com/marcoboers/home-assistant-quatt/releases/tag/0.5.0)

**Comment by marcoboers** [comment]
@Verguldebarman Sorry, I misunderstood. Fixed it in https://github.com/marcoboers/home-assistant-quatt/pull/84 with the new release [0.5.1](https://github.com/marcoboers/home-assistant-quatt/releases/tag/0.5.1)
